OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/KeychainService.java

```diff
@@ -653,7 +665,7 @@ public void setPreventCancel() {
     }
 
     public void serialKeyImport(ArrayList<ParcelableKeyRing> keyList, final String keyServer,
-                                ProviderHelper providerHelper) {
+                                ProviderHelper providerHelper, Proxy proxy) {
         Log.d(Constants.TAG, "serial key import starting");
         ParcelableFileCache<ParcelableKeyRing> cache =
                 new ParcelableFileCache<>(mKeychainService, "key_import.pcl");
```

Comment:
not directly related to this commit, but you should use KeychainService.this in these places instead of a local variable